cognite/client/testing.py

```diff
@@ -96,6 +98,9 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
 
         self.labels = MagicMock(spec_set=LabelsAPI)
 
+        self.models = MagicMock(spec=DataModelingAPI)
+        self.models.spaces = MagicMock(spec_set=SpacesAPI)
```

Comment:
This actually revealed a bug in `test_ensure_all_apis_are_available_on_cognite_mock` 🥳 . PR coming tomorrow.
```suggestion
        self.data_modeling = MagicMock(spec=DataModelingAPI)
        self.data_modeling.spaces = MagicMock(spec_set=SpacesAPI)
```

Reply:
https://github.com/cognitedata/cognite-sdk-python/pull/1234

Reply:
Congratulations, you got the 1234 :)